integration-test.py

```diff
@@ -311,6 +311,25 @@ def assert_at_least_n_features(z, x, y, layer, properties, n):
                 "%r, found only %d" % (n, properties, num_matching)
 
 
+def assert_less_than_n_features(z, x, y, layer, properties, n):
+    """
+    Downloads a tile and checks that it contains less than `n` features which
+    match the given `properties`.
+    """
+    with features_in_tile_layer(z, x, y, layer) as features:
+        num_features, num_matching = count_matching(
+            features, properties)
+
+        if num_features >= n:
```

Comment:
I'm just going to remove the `num_features` check altogether and rely on the `num_matching` check. Does that make sense?